app/src/main/res/values/material_colors.xml

```diff
@@ -284,12 +286,13 @@
     <color name="md_grey_700" tools:keep="@color/md_grey_700">#616161</color>
     <color name="md_grey_800">#424242</color>
     <color name="md_grey_900" tools:keep="@color/md_grey_900">#212121</color>
+    <color name="md_grey_950" tools:keep="@color/md_grey_900">#111111</color>
     <color name="md_black_1000">#000000</color>
     <color name="md_white_1000">#ffffff</color>
 
     <!--blue grey-->
     <color name="md_blue_grey_50" tools:keep="@color/md_blue_grey_50">#ECEFF1</color>
-    <color name="md_blue_grey_100">#CFD8DC</color>
+    <color name="md_blue_grey_100" tools:keep="@color/md_blue_grey_200">#CFD8DC</color>
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/worker8/Pixels/blob/789375a3d12930c1d033e15226a8f133d12a2222/app/src/main/res/values/material_colors.xml#L295"></span>The resource <code>R.color.md_blue_grey_100</code> appears to be unused</td>
    </tr>
  </tbody>
</table>